### ISSUE TITLE
Change event name to be a label

### ISSUE THREAD

**Issue by HarryEMartland** [nodeshift/opossum-prometheus#14]
To make graphing of circuit breakers easier would it be possible to change the event name to be a label rather than the name of the metric?
This would allow generic graphs to be made in Grafana rather than every time a new circuit breaker is added we will have to update our graphs.
Thanks

**Comment by lholmquist** [comment]
do you have some example code that you could share so we can get a better understanding of the request?

**Comment by HarryEMartland** [comment]
Not so much code however I am happy to do a pull request if you think the below is ok. This will break existing graphs and alerts which make use of the current metrics but will allow for more generic graphs in the future. E.g. when a new circuit breaker is added the graphs automatically show data.

As the metrics stand they look like;
```
circuit_example_fallback
circuit_example_fire
circuit_example_reject
```

I am suggesting they be changed to something like;
```
circuit{service="example",event="fallback"}
circuit{service="example",event="fire"}
circuit{service="example",event="reject"}
```

That allows PromQL/Graphs like
`sum(circuit) by (event, service)`

**Comment by lholmquist** [comment]
thanks!  now published as 0.1.0
